spec/lib/support/transform_position_spec.rb

```diff
@@ -8,6 +8,18 @@ class ChessPieceDouble
   end
   let(:transform_position) { ChessPieceDouble.new }
 
-  pending '#to_coordinate', '(from position "ay")'
-  pending '#to_position', '(from coordinate [x,y])'
+  context '#to_coordinate' do
+    context 'from d4' do
```

Comment:
Consider to make it more compact, and fix some tips 

* use rather describe instead of context for classes, modules and methods
* if you have context for one example, and you do need to prepare anything before, just write example (you don't have surround every spec with the context)

```ruby
describe "#to_coordinate" do
  it { expect(transform_position.to_coordinate('d4')).to eql([4, 4]) }
end 

describe "#to_position" do
  it { expect(transform_position.to_position(3, 2)).to eql('c2') }
end
```